README.md

```diff
@@ -1,16 +1,21 @@
 # Ideabox
 
 ### Summary
-
+An interactive repository of wit and wisdom-- all customizable by the user!
 ### How it Works
-- 
+- Open "index.html" on the Terminal.
+- Construct a new idea, by filling out the title and body fields and pressing the "Save" button.
+- A new Idea card will be created and deployed which the user can favorite, by pressing the star icon, or delete by pressing
+the close button, at will.
+- The user may toggle between viewing all ideas freely, or only those which have been favorited by pressing the corresponding button in the left-hand column.
+- The user may search the list of ideas, for content in the title OR body and only the relevant idea cards will display. Clearing the search bar, returns the user to the full view.
```

Comment:
- User ideas are stored locally, and will continue to be available when the user reloads the page or returns at a later time.